packages/patternfly-4/react-core/src/components/Breadcrumb/BreadcrumbItem.d.ts

```diff
@@ -0,0 +1,14 @@
+import { SFC, HTMLProps, ReactNode, ReactType } from 'react';
+
+export interface BreadcrumbItemProps extends HTMLProps<HTMLDivElement> {
```

Comment:
I believe this should extend `HTMLProps<HTMLElement>` since the props are passed to `nav` and not to `div`.